org.bndtools.templates.osgi/resources/templates/integration-test/bnd.bnd

```diff
@@ -1,9 +1,9 @@
 # This is the version of JUnit that will be used at build time and run time
-junit: org.apache.servicemix.bundles.junit;version="[4.11,5)"
+junit: org.apache.servicemix.bundles.junit;version=latest
```

Comment:
@kriegfrj can you give advise for the templates how to use JUnit 5? 

Personally I am ok with 4 since it is so much simpler and smaller. However, you've been advocating JUnit 5 a lot so should the templates be migrated?

Reply:
@chrisrueger I think we should refer to junit as a macro so we do not have this information in all bndrun files.

Reply:
@pkriens 

> @chrisrueger I think we should refer to junit as a macro so we do not have this information in all bndrun files.

I have added a line like this `junit: org.apache.servicemix.bundles.junit` to `build.bnd` of the workspace template ([next door](https://github.com/bndtools/bndtools.workspace.min/pull/11)). 
This how it can be used like `${junit}` right?